docs/csharp/language-reference/operators/arithmetic-operators.md

```diff
@@ -161,10 +161,10 @@ You also can use the `-` operator for delegate removal. For more information, se
 
 The following list orders arithmetic operators starting from the highest precedence to the lowest:
 
-- Postfix increment `x++` and decrement `x--` operators.
-- Prefix increment `++x` and decrement `--x` and unary `+` and `-` operators.
-- Multiplicative `*`, `/`, and `%` operators.
-- Additive `+` and `-` operators.
+- Postfix increment `x++` and decrement `x--` operators
+- Prefix increment `++x` and decrement `--x` and unary `+` and `-` operators
+- Multiplicative `*`, `/`, and `%` operators
+- Additive `+` and `-` operators
```

Comment:
Updated punctuation in this file based on the [style guide](https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation)